### ISSUE TITLE
DuckieTV Windows goes all white

### ISSUE THREAD

**Issue by ggervais5** [SchizoDuckie/DuckieTV#1306]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Standalone

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
Nightly 202102252304

**What is your Operating System (Windows, Mac, Linux, Android)**
.Windows 10 Pro N Build 19042.867

**Describe the problem you are having and steps to reproduce if available**
After DuckieTV sits in my system tray overnight, opening it in morning (to check what it did the night before) shows a window with the Minimize, Maximize, Close buttons and the window title, but the window shows all white.  To fix it, I need to exit the tray icon and restart DuckieTV manually.  Then it is fine until the next overnight.

**Attach any DuckieTV statistics or Developer Console logs if available**
...

**Comment by garfield69** [comment]
This has something to do with the 3rd party run time environment NWJS that we use to allow us to run Dtv in a Standalone mode.
Your version 202102252304 uses NWJS 50.3 with Chromium 87

Upgrade your Dtv to 202103062304 or newer, which uses NWJS 51.1 with Chromium 88 and see if that makes a difference.
Report back your findings.
Thanks.

**Comment by ggervais5** [comment]
I updated to the newest nightly build 202103102304 and the issue happened when I rebooted my laptop.  Issues in the past were not related to reboots, or at least not restricted solely to reboots.  In shell:startup, I have a command to run the DuckieTV executable on startup.  Perhaps there is a timing issue, where there are dependent services that need to be started before DuckieTV can start up properly.  Is there a better way to have DuckieTV start up when Windows starts?

**Comment by ggervais5** [comment]
I think I found something.  I was using an AutoIt script to run DuckieTV.  In the script, the command line said to run the program minimized.  Inside DuckieTV, I already had turned on the setting for the application to start up minimized.  Once I removed the ‘minimized’ option in the AutoIt script, the DuckieTV starts up minimized,  because of the app setting, and when I click on the icon, I see the GUI as I would expect to.  That seems to explain the behavior at reboot.  The previous version showed the white screen without reboots.  I’ll see if the white screen returns.

 

Greg
